dev-lang/php/php-7.4.16.recipe

```diff
@@ -87,6 +100,9 @@ BUILD()
 	autoreconf
 	autoconf
 	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir \
+		--sysconfdir=$settingsDir/php7 \
```

Comment:
Change required to install the FPM binary in the correct place (build on R1B2 32bit):
```diff
-	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir \
-		--sysconfdir=$settingsDir/php7 \
+	runConfigure --omit-dirs "binDir sbinDir" ./configure \
+		--bindir=$commandBinDir \
+		--sbindir=$commandBinDir \
+		--sysconfdir=$settingsDir/php7 \
```

Reply:
I'll get this together this weekend.

Reply:
PS, have to tell also that I didn't use the resolv.h patch (as the header is part of Haiku and should be picked up in the first place)

Reply:
Yea, the resolv.h patch is a workaround for those that run nightly builds and likely b3 when it comes.

The resolv.h that is included with Haiku does get picked up, but it causes the build failure @extrowerk was having. It's missing  "res_ndestroy". As I haven't had time to look into fixing it at the Haiku level, I opted to fix it at the package level.

As a side note, it's identical to the workaround for #5772.

Reply:
I left the resolv.h fix in and fix the php-fpm binary location.